setup.py

```diff
@@ -292,65 +302,72 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'pygithub',
 ]
 google = [
+    # Google has very clear rules on what dependencies should be used. All the limits below
+    # follow strict guidelines of Google Libraries as quoted here:
+    # While this issue is open, dependents of google-api-core, google-cloud-core. and google-auth
+    # should preserve >1, <3 pins on these packages.
+    # https://github.com/googleapis/google-cloud-python/issues/10566
+    # Some of Google Packages are limited to <2.0.0 because 2.0.0 releases of the libraries
+    # Introduced breaking changes across the board. Those libraries should be upgraded soon
+    # TODO: Upgrade all Google libraries that are limited to <2.0.0
```

Comment:
Given v2.0 if the libraries _did_ introduce breaking changes across the board (or so I understand?), I wonder if this is a case where w _should_ keep the `<2` restriction?

Reply:
Yep . I think I kept all `< 2.0.0` where they were there and only removed the <3, <4 which are future-unknown-if-ever-happens-and-what-the-scope-of-breakage-will-be ones. Let me re-review if I did it right.

Reply:
Yep. All the `<2.0.0` were left. I only removed <3 and <4.

This was actually even announced and "warned" before by Google that 2.0.0 for all the libraries will be breaking (they changed the way how 2.0.0 libraries were generated (they are mostly automatically generated). So I do not *really* expect similar kind of breakage even in the future versions (and we do not even know if they will ever be 3s or 4s for those)